build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
 
     //Unit testing framework.
     testCompile 'junit:junit:4.12'
+    //testCompile "org.mockito:mockito-core:2.+"
```

Comment:
not sure... place holder for when we have to include mockito to do mocking?  @ndushay?